### PR TITLE
Changed the login branding

### PR DESCRIPTION
**Pull request by GamenHaaDee**
Created normal buttons for Forget Password and Signup https://cdn.discordapp.com/attachments/803559100234989568/1071895681251414067/image.png

### CODE REVIEW

**Comment by John-S4** [comment]
As per my comment on #782 

I think that these changes are something that should probably be done by changing the template in a theme folder, rather than by changing the default in the module.

I can see why people might want them as buttons like that, but I can also see why a lot of people would prefer them to be less bold and conspicuous.

File: src/modules/Page/html_client/mod_page_login.html.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends "layout_public.html.twig" %}
 
-{% block meta_title %}{{ 'Log in'|trans }}{% endblock %}
+{% block meta_title %}{{ guest.system_company.name }}{{ ' - Log in'|trans }}{% endblock %}
```
Comment (jaapmarcus) [inline src/modules/Page/html_client/mod_page_login.html.twig:3]: It should be come

`{{ guest.system_company.name }} - {{ 'Log in'|trans }}`

Instead

Comment (GamenHaaDee) [reply src/modules/Page/html_client/mod_page_login.html.twig]: Changed it. to the style like the others

**Comment by GamenHaaDee** [comment]
@John-S4 Changed the button back to normal.
@jaapmarcus Changed the branding

**Comment by jaapmarcus** [comment]
In that case we should update all the titles...

**Comment by John-S4** [comment]
Signup / Forgot password link visibility is improved by #894

**Comment by evrifaessa** [comment]
Closing for now. Feel free to reopen anytime.

**Pull request closed without merging**